resolutions/proposed/2020-03.md

```diff
@@ -0,0 +1,36 @@
+# TODO Group Steering Committee
+## Resolution 2020-03
```

Comment:
We'll need to change the number here as well and rename the file.

Reply:
Done.